invenio_records_marc21/translations/de/LC_MESSAGES/messages.po

```diff
@@ -21,24 +21,24 @@ msgstr ""
 
 #: invenio_records_marc21/services/schemas/access.py:55
 msgid "Embargo date must be in the future."
-msgstr ""
+msgstr "Sperrfrist muss in der Zukunft liegen."
```

Comment:
it is also possible to use embargo in the translation. Better: "Das Ende des Embargos muss in der  Zukunft liegen"